docs-md/developer-guide/test-and-debug/generate-custom-test-data.md

```diff
@@ -66,20 +66,17 @@ Use the following command to generate records from one of the predefined schemas
+| -------------------------------------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| ``bootstrap-server=<kafka-server>:<port>``   | localhost:9092                                      | IP address and port for the Kafka server to connect to.                                                                               |                                                      |
+| ``key-format=<key format>``                  | Kafka                                               | Format of generated record keys: ``avro``, ``json``, ``delimited`` or ``kafka``. Case-insensitive. Required by the ``schema`` option. |                                                      |
+| ``value-format=<value format>``              | JSON                                                | Format of generated record values: ``avro``, ``json``, or ``delimited``.                                                              | Case-insensitive. Required by the ``schema`` option. |
+| ``topic=<kafka topic name>``                 |                                                     | Name of the topic that receives generated records. Required by the ``schema`` option.                                                 |                                                      |
+| ``key=<name of key column>``                 |                                                     | Field to use as the key for generated records. Required by the ``schema`` option.                                                     |                                                      |
+| ``iterations=<number of records>``           | 1,000,000                                           | The maximum number of records to generate.                                                                                            |                                                      |
+| ``msgRate=<rate to produce in msgs/second>`` | -1 (unlimited, i.e. as fast as possible)            | The rate to produce messages at, in messages-per-second.                                                                              |                                                      |
+| ``propertiesFile=<path-to-properties-file>`` | ``<path-to-confluent>/etc/ksql/datagen.properties`` | Path to the ``ksql-datagen`` properties file.                                                                                         |                                                      |
+| ``schemaRegistryUrl``                        | http://localhost:8081                               | URL of {{ site.sr }} when ``format`` is ``avro``.                                                                                     |                                                      |
```

Comment:
main change here is switching the deprecated `maxInterval` for its replacement: `msgRate`